### PR TITLE
Ukraine, Bug Fixes, GUI Major Stuff, Finnish/Denmark errors

### PR DESCRIPTION
**Pull request by Thanasis09**
Only some loc may be missing

### CODE REVIEW

**Review by dax0102 — DISMISSED** [review]

File: common/characters/IRQ.txt
Comment (dax0102) [inline common/characters/IRQ.txt:1]: Merge Faisal into one character

Comment (Thanasis09) [reply common/characters/IRQ.txt:1]: i don't have an on action rn, i would need to make one

Comment (Thanasis09) [reply common/characters/IRQ.txt:1]: neither a focus or something to change portrait

File: events/FX_Greece.txt [hunk 1]
```diff
@@ -30,12 +30,12 @@ country_event = {
 				id = gre.1
 			}
 			GRE = {
-				add_ideas = {
-					GRE_Eleftherios_Kyriakou_Venizelos_hog_sli
-					GRE_Andreas_Michalakopoulos_for_sli
-					GRE_Alexandros_Diomidis_eco_sli
-					GRE_Georgios_Maris_sec_sli
-				}
+				# add_ideas = {
```
Comment (dax0102) [inline events/FX_Greece.txt:33]: Remove the comments

File: history/countries/GRE - Greece.txt [hunk 2]
```diff
@@ -386,3 +387,9 @@ recruit_character = GRE_Stylianos_Lykoudis
 recruit_character = GRE_Dimitrios_Gounaris
 recruit_character = GRE_Konstantinos_Tsaldaris
 recruit_character = GRE_Konstantinos_Karamanlis
+
+# 🖕🖕🖕🖕 this also doesn't work, fuck paradox 🖕🖕🖕
```
Comment (dax0102) [inline history/countries/GRE - Greece.txt:391]: remove the comments

**Review by dax0102 — APPROVED** [review]

**Comment by Thanasis09** [comment]
Don't delete branch incase revert is needed

**Pull request merged**